### PR TITLE
docs: Project Orgnization

### PR DESCRIPTION
**Pull request by RobertGemmaJr**
- Adds "Project Organization" dropdown grouping
- Renames `project_organization` as `directory_structure` and updates it as needed
- Adds a table of the environment variables on `environment_variables`
- Updates `npm_scripts` as needed

### CODE REVIEW

**Comment by RobertGemmaJr** [comment]
This is what the table looks like

image

File: docs/project_organization/environment_variables.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+| `EVENT_MARKER_PRODUCT_ID`  | ![desktop](https://img.shields.io/badge/desktop-yellow)                                                       | ![run](https://img.shields.io/badge/run-purple)   | ""            | ![string](https://img.shields.io/badge/string-grey)        | The product ID of the event marker. If not set, it will default to the `productID` set in `public/config/trigger.js`. If neither are set it will attempt to connect using the COM name. |
+| `EVENT_MARKER_COM_NAME`    | ![desktop](https://img.shields.io/badge/desktop-yellow)                                                       | ![run](https://img.shields.io/badge/run-purple)   | "COM3"        | ![string](https://img.shields.io/badge/string-grey)        | The COM name of the event marker. If not set, it will use the `comName` set in `public/config/trigger.js`. Field is ignored if `EVENT_MARKER_PRODUCT_ID` is set.                        |
```
Comment (digicosmos86) [inline docs/project_organization/environment_variables.mdx]: Love this table!

File: docs/project_organization/npm_scripts.mdx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+- `start` runs the app in development mode and does NOT open a browser. This makes working with electron easier as it creates its own browser for development.
+- `start:browser` runs the app in development mode and automatically opens a new browser tab. It uses your system's default browser. The page will reload if you make edits.
+- `start:electron`: Waits for the app to be running (from `start`) and then starts an electron process.
+
+## Firebase
+
+- `firebase:emulators:start` starts the Firebase emulators with the data found in `emulator_data/`
+- `firebase:emulators:save` saves the current state of the Firebase emulators into `emulator_data/`
+- `firebase:download` runs the `firebase-download-script.js` script, saving certain data in Firebase to your local machine.
```
Comment (digicosmos86) [inline docs/project_organization/npm_scripts.mdx:26]: Do we have an upload script somewhere as well?

File: docs/project_organization/npm_scripts.mdx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+- `dev:home:video` runs the app with equipment disabled and video enabled
+- `dev:clinic` runs the app with equipment enabled
+- `dev:clinic:video` runs the app with equipment enabled and video enabled
+- `dev:turk-prolific` runs the app with prolific and PsiTurk enabled
+
+### Firebase
+
+Working with Firebase is slightly different as it is meant to be run on the browser. `npm run dev:firebase` executes `start:browser` with Firebase enabled, which will automatically launch the running app in your default browser. It can be found on other browsers by navigating to [localhost:3000](https://localhost:3000).
+
+`dev:firebase` must be run in conjunction with `firebase:emulators:start` to work properly. This runs the Firebase Emulators locally with some dummy data. The emulator data can be viewed at [localhost:4000](https://localhost:4000).
```
Comment (digicosmos86) [inline docs/project_organization/npm_scripts.mdx:44]: This is great! Very clear

File: docs/deployments/firebase.mdx [hunk 4]
```diff
@@ -159,7 +159,7 @@ Honeycomb populates the Firestore emulators with the study `s1` and participant
 
 ## Deploying on Firebase
 
-Firebase deployments are handled automatically following [Continuous Integration Continuous Development](ci) practices using GitHub Actions. Here we will connect create custom actions that are connected to the task's Firebase project.
+Firebase deployments are handled automatically following [Continuous Integration Continuous Development](ci_cd) practices using GitHub Actions. Here we will connect create custom actions that are connected to the task's Firebase project.
```
Comment (digicosmos86) [inline docs/deployments/firebase.mdx]:
```suggestion
Firebase deployments are handled automatically following [Continuous Integration Continuous Development (CI/CD)](ci_cd) practices using GitHub Actions. Here we will connect create custom actions that are connected to the task's Firebase project.

```

**Review by digicosmos86 — APPROVED** [review]
Looking great! Much clearer now

**Pull request merged**